src/Forms/Api/reducers.js

```diff
@@ -3,6 +3,13 @@ import {
     UPDATE_USER_FORM,  
 } from './actions';
 
+// export const userInitialState = {
+//     user: {
+//         firstName: '',
+//         lastName: '',
+//         email: '',
+//     }
+// }
```

Comment:
Do we need this commented code?

Reply:
As of how things are taking shape with the form, no we probably will not need this snippet of code. I left it in though just incase it comes out that it will be better for the BE if I pull from the 'form store' and then replace the info into that userInitialState to then send up to you guy. It was jut a thought though.